admin-ui/src/components/ui/tooltip.tsx

```
@@ -0,0 +1,30 @@
+import * as React from "react"
+import * as TooltipPrimitive from "@radix-ui/react-tooltip"
```

ESLint

[error] Expected "@radix-ui/react-tooltip" to come before "react".

(perfectionist/sort-imports)